app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Admin::BaseController < ApplicationController
```

Comment:
I liked the exploration we did in figuring out whether we could use 1 method or 2.